pytket/docs/changelog.rst

```diff
@@ -4,6 +4,13 @@ Changelog
 1.8.1 (November 2022)
 ---------------------
 
+Minor new features:
+
+* Added shortcuts for adding ``U1``, ``U2``, ``U3``, ``TK1``, ``TK2``, ``CU1``, 
+  ``CU3``, ``ISWAP``, ``PhasedISWAP``, ``ESWAP``, ``PhasedX``, ``FSim``, ``Sycamore``
+  and ``ISWAPMax`` gates to a ``pytket`` ``Circuit``.
+* New ``Circuit`` methods ``n_1qb_gates``, ``n_2qb_gates``, ``n_nqb_gates``.
```

Comment:
These should be under 1.9.0 (unreleased).

Reply:
done